apisix/cli/ngx_tpl.lua

```diff
@@ -114,6 +114,27 @@ http {
             stub_status;
         }
     }
+
+    {% else %}
+
+    init_by_lua_block {
```

Comment:
1. this block is for `stream` only, please take care of the `http`
2. init_by_lua_block should be kept

Reply:
> 1. this block is for `stream` only, please take care of the `http`
> 2. init_by_lua_block should be kept

I have made a corresponding modification based on your two modification opinions